src/GitVersion.Core/GitVersion.Core.csproj

```diff
@@ -15,6 +15,7 @@
         <PackageReference Include="Polly" />
         <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
         <PackageReference Include="Microsoft.Extensions.Options" />
+        <PackageReference Include="YamlDotNet" />
```

Comment:
We should no include `YamlDotNet`, we just removed a couple of releases ago, so if you need anything configuration related do it in Configuration module